controllers/podplacement/pod_model.go

```diff
@@ -397,6 +399,17 @@ func (pod *Pod) isNodeSelectorConfiguredForArchitecture() bool {
 	return true
 }
 
+// isPodFromDaemonSet returns true if the pod is from a daemonSet.
+func (pod *Pod) isFromDaemonSet() bool {
+	// Check all ownerRef
+	for _, ownerRef := range pod.OwnerReferences {
+		if ownerRef.Kind == "DaemonSet" && ownerRef.Controller != nil && *ownerRef.Controller {
```

Comment:
```suggestion
		if ownerRef.Kind == "DaemonSet" && ownerRef.Controller != nil && *ownerRef.Controller {
```

is there a constant we can use in the apimachinery or similar k8s libs?

Reply:
not from a quick look.